generators/frontend/templates/react-demo/components/Features.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faStar, faHandSpock, faShippingFast, faHeart } from '@fortawesome/fontawesome-free-solid'
+
+export default class extends Component {
+  render() {
+    const Wrapper = styled.div`
+      display: flex;
+      max-width: 82em;
+      flex-direction: column;
+      text-align: center;
+      flex-wrap: wrap;
+      margin: 0 auto;
+      h1 {
+        font-size: 20px;
+        color: darkslategrey;
+        text-transform: uppercase;
+      }
+      div {
+        display: inline-flex;
+        margin-top: 20px;
+      `
+    const Feature = styled.div`
+      display: flex;
+      flex-direction: column;
+      width: 32em;
+      margin: 0 30px;
+      line-height: 28px;
+      color: darkslategrey;
+        .icon {
+          margin: 10px auto 10px auto;
+          margin-bottom: 20px;
+          justify-content: center;
+        }
+        h1 {
+          margin: 0;
+          text-align: center;
+          font-size: 16px;
+          font-weight: 600;
+          color: darkslategrey;
```

Comment:
You need to consider to pickup color convention for yourself.
I would recommend to use hex, unless you need to use opacity, then use rgba.
Currently you use everything: hex and rgba and color name ;)